build: Don't install @edx/studio-frontend

feanil opened the pull request:
The dependencies on this package via studio should all be removed now
and so we no longer need to install this package to pickup any
components from it.

This work is part of:
* https://github.com/openedx/edx-platform/issues/36275
* https://github.com/openedx/edx-platform/issues/36108

feanil committed:
build: Don't install @edx/studio-frontend

The dependencies on this package via studio should all be removed now
and so we no longer need to install this package to pickup any
components from it.

This work is part of:
* https://github.com/openedx/edx-platform/issues/36275
* https://github.com/openedx/edx-platform/issues/36108

open-craft-grove commented:
Sandbox deployment successful 🚀
🎓 [LMS](https://pr-37484-139931.sandboxes.opencraft.hosting)
📝 [Studio](https://studio.pr-37484-139931.sandboxes.opencraft.hosting)
ℹ️ [Grove Config](https://axim-sandbox-build-logs-4626178240.nyc3.digitaloceanspaces.com/pr-37484-139931-51564656-11717034807-tutor-config.yml), [Tutor Config](https://axim-sandbox-build-logs-4626178240.nyc3.digitaloceanspaces.com/pr-37484-139931-51564656-11717034807-grove-config.yml), [Tutor Requirements](https://axim-sandbox-build-logs-4626178240.nyc3.digitaloceanspaces.com/pr-37484-139931-51564656-11717034807-tutor-requirements.txt)

feanil commented:
There is still one component in the old unit editor that uses a component from studio-frontend, so we can't move forward with this one until https://github.com/openedx/edx-platform/issues/35261 is done.

feanil committed:
fix: Drop other references to studiofrontend.

Drop tooling to load studio-frontend components into mako templates and
XSS testing features related to it.

feanil commented:
The last component that was using the studio frontend has now been removed also so this should be safe to land!

kdmccormick reviewed (approved):
Pending a green build and some smoke testing, this looks good to me! 🔥 🔥 🔥

open-craft-grove commented:
Sandbox deployment successful 🚀
🎓 [LMS](https://pr-37484-139931.sandboxes.opencraft.hosting)
📝 [Studio](https://studio.pr-37484-139931.sandboxes.opencraft.hosting)
ℹ️ Configuration information unavailable

open-craft-grove commented:
Sandbox deployment successful 🚀
🎓 [LMS](https://pr-37484-139931.sandboxes.opencraft.hosting)
📝 [Studio](https://studio.pr-37484-139931.sandboxes.opencraft.hosting)
ℹ️ Configuration information unavailable

feanil commented:
Smoke testing looks good, I'll merge this later today after the last PR it depends on has landed on `master`.
